sdk/appconfiguration/Azure.Data.AppConfiguration/tests/ConfigurationMockTests.cs

```diff
@@ -472,12 +472,12 @@ await foreach (ConfigurationSetting value in service.GetConfigurationSettingsAsy
 
             MockRequest request1 = mockTransport.Requests[0];
             Assert.AreEqual(RequestMethod.Get, request1.Method);
-            Assert.AreEqual($"https://contoso.appconfig.io/kv/?key=%2A&label=%2A&api-version={s_version}", request1.Uri.ToString());
+            Assert.AreEqual($"https://contoso.appconfig.io/kv/?api-version={s_version}", request1.Uri.ToString());
```

Comment:
Why did this URI change?

Reply:
Both `key` and `label` are optional. We don't need to supply them to select everything.